Makefile

```diff
@@ -1,6 +1,18 @@
-WEBROOT=~/src/github.com/kubernetes/website
-K8SROOT=~/k8s/src/k8s.io/kubernetes
-MINOR_VERSION=17
+#
+# To generate docs with Makefile targets
+# from this repo base directory,
+# set the following environment variables
+# to match your environment
+#
+# WEBROOT=~/src/github.com/kubernetes/website
+# K8SROOT=~/k8s/src/k8s.io/kubernetes
+# K8S_RELEASE=17
+
+WEBROOT=${K8S_WEBROOT}
+K8SROOT=${K8S_ROOT}
+MINOR_VERSION=${K8S_RELEASE}
```

Comment:
I was expecting that the ${K8S_RELEASE} be a full version string like `1.17`.

Reply:
@tengqm , I could change the value to be the `major.minor` version. Currently, the kubectl-command directories follow a pattern such as:
v1_17. Could drop the `v` in the directory name.

Reply:
A couple of thoughts:
- kubectl is not versioned in k8s/website
- api ref is versioned in k8s/website, but creates a directory like `v1.17` instead of `v1_17`
- the makefile is more complicated. Could introduce a versioning config that saves two vars:
latest and prev. Use these variables for copying config files from release to release.